wildfly/connectors/webservice/connector-ws/pom.xml

```diff
@@ -144,8 +144,8 @@
             <artifactId>spring-tx</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.jboss.spec.javax.xml.bind</groupId>
-            <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+            <groupId>org.glassfish.jaxb</groupId>
```

Comment:
This needs changed to the generic jaxb instead right?

Reply:
replaced with javax.xml.bind